code/outils.sls

```diff
@@ -0,0 +1,142 @@
+
+(library (outils)
+  (export lookup
+          lookup-partial
+          lookup-spine
+          
+          ;; read/write utilities
+          read-all
+          write-expression-to-file
+          write-r6rs-expression-to-file
+          save-fasl
+          fasl-load
+
+          ;; config utilities
+          processed-config
+          persist-config
+          load-config
+
+          ;; configlet utilities
+          configlet-uuid)
+  (import (chezscheme))
+
+  ;;; Assoc lists
+
+  (define lookup
+    (case-lambda
+      ((symbol)
+       (lambda (thing)
+         (lookup symbol thing)))
+      ((symbol thing)
+       (cond ((assoc symbol thing) => cdr)
+             (error 'lookup "key not in alist" symbol thing)))
+      ((default symbol thing)
+       (cond ((assoc symbol thing)
+              => cdr)
+             (else default)))))
```

Comment:
Looks like you already updated `lookup` to handle the work of `lookup-partial` are you planning on removing calls to `lookup-partial` so it can be dropped?

Reply:
Other than wondering about the persistence of `lookup-partial` I am happy as could be about this :smile: